lib/ts/recipe/thirdparty/index.ts

```diff
@@ -172,6 +172,8 @@ export default class Wrapper {
     static Facebook = Facebook;
     static Github = Github;
     static SignInAndUp = (prop?: any) => ThirdParty.getInstanceOrThrow().getFeatureComponent("signinup", prop);
+    static SignInAndUpNoRedirect = (prop?: any) =>
+        ThirdParty.getInstanceOrThrow().getFeatureComponent("signinup-noredirect", prop);
```

Comment:
Should we just not exposed a redirect version of that at all? Will be easier to do docs.. otherwise we have to explain somewhere that there are two versions.

Reply:
I think the main use case for that is if devs just want to change the path and/or wrap it in some kind of layout. This seems a bit more common than the no-redirect one. Still, it'd be easier to document.
Another option is to expose them as a single version but add an extra prop that controls if a redirect should happen.